cypress/integration/features/account/changeLanguage.feature

```diff
@@ -0,0 +1,49 @@
+@account @stable_test
+Feature: Account - Change language of the user from user setting
+
+    As a user, I want to language change from user menu for all users i.e. (Admin, Teacher, Student)
+
+    Scenario Outline: Teacher can change language to '<language>'
+        Given I am logged in as a 'teacher' at 'brb'
+        When I arrive on the dashboard
+        Then I can see initials of my name
+        When I click on initials of my name
+        Then I click on language drop down menu
+        Then I can change language to '<language>'
+        Then I can see language is changed to '<language>'
```

Comment:
If we'd call this step "Then I see title of dashboard now is in <language>" it would be more clear what exactly is checked here.

Reply:
I'll update it, thx